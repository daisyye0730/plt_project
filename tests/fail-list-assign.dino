int main() {
    List(string, 3) li;
    li = ["hello", 5, 'c'];
}