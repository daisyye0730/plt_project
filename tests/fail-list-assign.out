Fatal error: exception Failure("List elements not uniform")
